Loaded dependency [python3/3.9.11]: gcc/10.3.0-binutils-2.36.1

Switching from python3/3.8.2 to python3/3.9.11
  Loading requirement: gcc/10.3.0-binutils-2.36.1
2023-09-28 23:29:19.840505: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
I0000 00:00:1695936575.993696    8358 tfrt_cpu_pjrt_client.cc:349] TfrtCpuClient created.
No GPU/TPU found, falling back to CPU. (Set TF_CPP_MIN_LOG_LEVEL=0 and rerun for more info.)
2023-09-28 23:29:39.575432: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1960] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...
/zhome/77/8/118225/.local/lib/python3.9/site-packages/numpy/core/getlimits.py:518: UserWarning: The value of the smallest subnormal for <class 'numpy.float32'> type is zero.
  setattr(self, word, getattr(machar, word).flat[0])
/zhome/77/8/118225/.local/lib/python3.9/site-packages/numpy/core/getlimits.py:89: UserWarning: The value of the smallest subnormal for <class 'numpy.float32'> type is zero.
  return self._float_to_str(self.smallest_subnormal)
Traceback (most recent call last):
  File "/zhome/77/8/118225/Desktop/smrdm/smrdm/train_score.py", line 2286, in <module>
    train_sn_s1(file_path='models/S1', 
  File "/zhome/77/8/118225/Desktop/smrdm/smrdm/train_score.py", line 1030, in train_sn_s1
    train_s1(M=M,
  File "/zhome/77/8/118225/Desktop/smrdm/smrdm/jaxgeometry/statistics/score_matching/trainxt.py", line 140, in train_s1
    np.save(os.path.join(save_path, "loss_arrays.npy"), jnp.stack(loss))
  File "<__array_function__ internals>", line 200, in save
  File "/zhome/77/8/118225/.local/lib/python3.9/site-packages/numpy/lib/npyio.py", line 518, in save
    file_ctx = open(file, "wb")
FileNotFoundError: [Errno 2] No such file or directory: 'models/S12/dsm/loss_arrays.npy'
terminate called without an active exception
/zhome/77/8/118225/.lsbatch/1695850151.17891474.shell: line 21:  8358 Aborted                 python3 train_score.py --model S1 --order s1
